osbuild/meta.py

```diff
@@ -585,3 +629,50 @@ def get_schema(self, klass, name=None, version="1") -> Schema:
         self._schemata[(klass, name, version)] = schema
 
         return schema
+
+    def list_runners(self, distro: Optional[str] = None) -> List[RunnerInfo]:
+        """List all available runner modules"""
+        if not self._runners:
+            path = os.path.join(self.path, "runners")
+            names = filter(lambda f: os.path.isfile(f"{path}/{f}"),
+                           os.listdir(path))
+            paths = map(lambda n: os.path.join(path, n), names)
+
+            runners_mapped = map(RunnerInfo.from_path, paths)
+            runners_sorted = sorted(runners_mapped, key=lambda r: r.distro)
+            runners_sorted = sorted(runners_sorted, key=lambda r: r.version)
+
```

Comment:
You can turn this sort into a single one: `runners_sorted = sorted(runners_mapped, key=lambda r: (r.distro, r.version))`. I'm not sure on the stability of doing sorted twice like this.

Reply:
Maybe there could be a unit test to check that the order is the one wanted.